GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -1883,26 +1883,19 @@ private void Stage(IReadOnlyList<GitItemStatus> items)
                     }
 
                     bool wereErrors = false;
-                    if (AppSettings.ShowErrorsWhenStagingFiles)
```

Comment:
I am not convinced this check should be removed, other errors should be possible

Reply:
I just checked the [documentation](https://git-extensions-documentation.readthedocs.io/en/latest/settings.html#git-extensions-detailed-commit-dialog):
> Show errors when staging files
If an error occurs when files are staged (in the Commit dialog), then the process dialog showing the results of the git command is shown if this setting is checked.

Since it speaks generally of "an error" I guess the real intention of this flag is "any kind of error", not only those specific to line endings. So you're right that this check should not be removed.

Reply:
Done.